### PR TITLE
WIP: Directory View and Project View Components

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
The scope of this PR will include much of the functionality for the my projects view (almost all of it), but will strictly be the directory view with filtering, sorting, etc

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/data.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const mapRecordsToProps = (props: IProps) => {
+  const {
+    sortProperty,
+    defaultSort,
+    filterOptions,
+    pageOptions,
+  } = props;
+
+  return {
+    q => q
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/data.tsx]: jsonapi is the best :heart: :heart_eyes:

File: source/SIL.AppBuilder.Portal.Frontend/src/data/containers/sorting.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/data/containers/sorting.tsx]: Not sure we need to redo our own component. Semantic table can do sorting.

https://react.semantic-ui.com/collections/table#variations-sortable

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/data/containers/sorting.tsx]: I need to hook in to orbit, so I'll be ignoring semantic's 'features' :)

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx]: this looks like it returned from the dead? I don't think we need this file?

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx]: @giancorzo

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx]: I'm going to merge master to this branch that would fix it

**Pull request merged**